src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -0,0 +1,875 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
```

Comment:
Sure, will do. I  do have a TODO item tracking the work to add tests for scoping rules. But adding one test right now sounds reasonable. 

---
In reply to: [63750290](https://github.com/dotnet/roslyn/pull/11393#discussion_r63750290) [](ancestors = 63750290)